pkg/crc/machine/types/types.go

```diff
@@ -46,12 +46,22 @@ type StopResult struct {
 
 type ClusterStatusResult struct {
 	CrcStatus        state.State
-	OpenshiftStatus  string
+	OpenshiftStatus  OpenshiftStatus
 	OpenshiftVersion string
 	DiskUse          int64
 	DiskSize         int64
 }
 
+type OpenshiftStatus string
+
+const (
+	OpenshiftUnreachable OpenshiftStatus = "Unreachable"
+	OpenshiftStarting    OpenshiftStatus = "Starting"
+	OpenshiftRunning     OpenshiftStatus = "Running"
+	OpenshiftDegraded    OpenshiftStatus = "Degraded"
+	OpenshiftStopped     OpenshiftStatus = "Stopped"
+)
+
```

Comment:
@guillaumerose  I want to know what is the benefit we are going to have with enum and  if we do something like following? We are only making sure it is only single place. Even with enum if someone want to change the value they can can in the code right?

```
const (
	OpenshiftUnreachable  = "Unreachable"
	OpenshiftStarting     = "Starting"
	OpenshiftRunning      = "Running"
	OpenshiftDegraded     = "Degraded"
	OpenshiftStopped      = "Stopped"
)
```

Reply:
For instance, OpenshiftStopped is used twice and it is easy to find where. 
Before this enum, I would have to search for "Stopped" string. I would get tons of result. 

It is also a good way to show consumers what are the possible values. 

The point of this PR is also to highlight these 3 enums and do something with that. 
I don't know what but there is something to do for sure!